### PR TITLE
[Coding Style] Enable rule PSR2.Classes.PropertyDeclaration.ScopeMissing

### PR DESCRIPTION
**Pull request by michalkleiner**
### Description:

Please include a description of this change and which issue it fixes. If no issue exists yet please include context and what problem it solves.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by michalkleiner** [comment]
I've renamed a few variables to remove the leading underscore from them. Where it made sense I added a prefix, in some instances I just removed the underscore. Arguably for the public visibility variables this could be seen as a BC breakage but in tests and in other instances changed here I'd say it's fine as it's not a part of our public API in its current definition.

File: plugins/API/API.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ class API extends \Piwik\Plugin\API
      * For Testing purpose only
      * @var int
      */
-    public static $_autoSuggestLookBack = 60;
```
Comment (sgiehl) [inline plugins/API/API.php]: This one is used in various plugins, so we actually can only change that in a major release.

File: core/EventDispatcher.php [hunk 2]
```diff
@@ -60,7 +60,7 @@ public static function getInstance()
 
     private $pluginHooks = array();
 
-    public static $_SKIP_EVENTS_IN_TESTS = false;
+    public static $skipEventsInTests = false;
```
Comment (sgiehl) [inline core/EventDispatcher.php]: This property could in theory also be used by plugins in their tests. I'm not aware of a plugin, that is doing it, but we shouldn't change that silently in a patch release.

**Review by sgiehl — CHANGES_REQUESTED** [review]
Even though you only renamed variables used for testing purpose, this is a breaking change. As that stuff might be in use by plugins for their tests, we shouldn't change that in a coding style change.

**Comment by michalkleiner** [comment]
Yeah, I wasn't 100% sure what the best course of action here would be. Should we just close the PR for now? I'd be ok with that.

**Comment by sgiehl** [comment]
Guess the rule `PSR2.Classes.PropertyDeclaration.ScopeMissing` can be applied. But the underscore rules need to wait...

File: plugins/API/API.php (hunk 1 above)
Comment (michalkleiner) [reply plugins/API/API.php]: Reverted the underscore rule.

File: core/EventDispatcher.php (hunk 2 above)
Comment (michalkleiner) [reply core/EventDispatcher.php]: Reverted the underscore rule.

**Comment by michalkleiner** [comment]
Changed the PR to just enable the `PSR2.Classes.PropertyDeclaration.ScopeMissing` rule.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**